meta-k8s-setup/recipes-k8s-configuration/k8s-configuration/files/crio.service

```
@@ -1,9 +1,9 @@
 [Unit]
 Description=Container Runtime Interface for OCI (CRI-O)
 Documentation=https://github.com/cri-o/cri-o
-Wants=network.target
+Wants=network.target k8s-configuration.service
```

```suggestion
Wants=network.target
```
Let's not do this. `k8s-configuration.service` shouldn't be pulled in by some "random" service. Let's pull it in with the mount unit.